index.html

```diff
@@ -1,16 +1,113 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css" />
     <title>Responsive Cake webpage</title>
+    <script
+      src="https://kit.fontawesome.com/02e465af65.js"
+      crossorigin="anonymous"
+    ></script>
     <!-- Add a link to your css file here -->
-</head>
+  </head>
+
+  <body>
+    <header class="header">
+      <div class="header-container">
+        <a class="header-img"><img src="/img/logo.jpg" /></a>
```

Comment:
Logo image is missing. Try `src="img/logo.jpg" ` instead :)

Reply:
amended pathway of logo source.